help.c

```diff
@@ -77,24 +89,32 @@ static const struct MenuFuncOp *help_lookup_function(int op, enum MenuType menu)
 }
 
 /**
- * print_macro - Print a macro string to a file
- * @param[in]  fp       File to write to
- * @param[in]  maxwidth Maximum width in screen columns
- * @param[out] macro    Macro string
- * @retval num Number of screen columns used
+ * help_sort_alpha - Compare two Help Lines by their first entry - Implements ::sort_t - @ingroup sort_api
+ */
+static int help_sort_alpha(const void *a, const void *b, void *sdata)
+{
+  struct HelpLine x = *(const struct HelpLine *) a;
+  struct HelpLine y = *(const struct HelpLine *) b;
+
+  return mutt_str_cmp(x.first, y.first);
```

Comment:
How about avoiding a copy:

```suggestion
  const struct HelpLine *x = (const struct HelpLine *) a;
  const struct HelpLine *y = (const struct HelpLine *) b;

  return mutt_str_cmp(x->first, y->first);
```